nativelib/src/main/scala/scala/scalanative/native/alwaysinline.scala

```diff
@@ -0,0 +1,5 @@
+package scala.scalanative
+package native
+
+/** Unconditionally inline annotated method. */
+final class alwaysinline extends scala.annotation.StaticAnnotation
```

Comment:
That's interesting. We treat `@inline` as optional hint (`inlinehint` in terms of LLVM), similarly to how clang works for inline in C++. Do you think that we should do the same as SJS here instead?